src/coinbase/wallet.ts

```diff
@@ -60,17 +67,24 @@ export class Wallet {
    * Instead, use User.createWallet.
    *
    * @constructs Wallet
-   * @param intervalSeconds - The interval at which to poll the CDPService, in seconds.
-   * @param timeoutSeconds - The maximum amount of time to wait for the ServerSigner to create a seed, in seconds.
+   * @param options - The options to create the Wallet.
+   * @param options.networkId - the ID of the blockchain network. Defaults to 'base-sepolia'.
+   * @param options.intervalSeconds - The interval at which to poll the CDPService, in seconds.
+   * @param options.timeoutSeconds - The maximum amount of time to wait for the ServerSigner to create a seed, in seconds.
    * @throws {ArgumentError} If the model or client is not provided.
    * @throws {InternalError} - If address derivation or caching fails.
    * @throws {APIError} - If the request fails.
    * @returns A promise that resolves with the new Wallet object.
    */
-  public static async create(intervalSeconds = 0.2, timeoutSeconds = 20): Promise<Wallet> {
+  public static async create(options?: WalletCreateOptions): Promise<Wallet> {
+    const {
+      networkId = Coinbase.networkList.BaseSepolia,
+      intervalSeconds = 0.2,
+      timeoutSeconds = 20,
+    } = options || {};
```

Comment:
this is inconsistent with how we are passing options types in our `Coinbase` constructors in `.../coinbase.ts`. 
Ref: https://github.com/coinbase/coinbase-sdk-nodejs/blob/fadd3f23042c5fa81f862a9bdc8467ec8ea19c64/src/coinbase/coinbase.ts#L74

and 

https://github.com/coinbase/coinbase-sdk-nodejs/blob/fadd3f23042c5fa81f862a9bdc8467ec8ea19c64/src/coinbase/coinbase.ts#L119

Lets choose one and stay consistent